src/components/Editor/index.js

```diff
@@ -166,7 +166,11 @@ class Editor extends PureComponent<Props, State> {
         return this.onGutterContextMenu(event);
       });
 
-      codeMirror.on("contextmenu", (cm, event) => this.openMenu(event));
+      codeMirror.on("contextmenu", (cm, event) => {
+        event.stopPropagation();
+        event.preventDefault();
+        return this.openMenu(event);
```

Comment:
minor point, but for aesthetics can we move `event.stopPropagation` and `event.preventDefault` to `openMenu`. It'll only ever be called here